.gitlab-ci.yml

```diff
@@ -0,0 +1,161 @@
+image: alpine:latest
+
+variables:
+  APP_NAME: nakadi
```

Comment:
APP_NAME "nakadi" amend to host name. Since shall we use the APP_NAME: event-hub

Reply:
Addressed in 16e5e0dd19d17688366991627ccf4cadefd0ce34